vagrant/deploy.sh

```diff
@@ -7,9 +7,8 @@ PROJECT_DIR=$(cd "${SCRIPT_DIR}/.." && pwd)
 cd "$SCRIPT_DIR"
 
 # Load environment variables
-export TARGET_CONCOURSE=bootstrap
 # shellcheck disable=SC2091
-$("${PROJECT_DIR}/concourse/scripts/environment.sh")
+$("${PROJECT_DIR}/concourse/scripts/environment.sh" "$@")
```

Comment:
You must add here `eval` before, or otherwise you get the error:

```
$ BRANCH=$(git rev-parse --abbrev-ref HEAD) DEPLOY_ENV=hector make dev deployer-concourse bootstrap 
vagrant/deploy.sh
vagrant/deploy.sh: line 11: export: `#': not a valid identifier
vagrant/deploy.sh: line 11: export: `WARNING:': not a valid identifier
vagrant/deploy.sh: line 11: export: `paas-bootstrap': not a valid identifier
make: *** [bootstrap] Error 1
```

You can check it by running:

```
$(DEPLOY_ENV=hector /home/keymon/gds/paas-bootstrap/concourse/scripts/environment.sh)
``` 
vs 

``` 
eval $(DEPLOY_ENV=hector /home/keymon/gds/paas-bootstrap/concourse/scripts/environment.sh)
```